src/components/Form.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import './form.css';
+
+export default function Form({
+  name,
+  age,
+  bio,
+  image,
+  breed,
+  setName,
+  setAge,
+  setBio,
+  setImage,
+  setBreed,
+  save,
+  success,
+}) {
+  return (
+    <form>
+      <p>{success}</p>
```

Comment:
message might make a little more sense here, as this message shows both with a success and failure